client/my-sites/plans/main.jsx

```diff
@@ -305,6 +317,24 @@ class Plans extends Component {
 		return <ECommerceTrialPlansPage interval={ interval } site={ selectedSite } />;
 	}
 
+	renderBusinessTrialPage() {
+		const { selectedSite, currentPlan } = this.props;
+
+		if ( ! selectedSite ) {
+			return this.renderPlaceholder();
+		}
+
+		const interval = this.getIntervalForBusinessMigrationPlans();
+
+		return (
+			<BusinessTrialPlansPage
+				currentPlan={ currentPlan }
+				interval={ interval }
```

Comment:
Do we need `currentPlan` and `interval` here? I don't see it gets used in the component.